recipes/rfc3986-validator/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "rfc3986-validator" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/9e/51/17023c0f8f1869d8806b979a2bffa3f861f26a3f1a66b094288323fba52f/rfc3986_validator-{{ version }}-py2.py3-none-any.whl
+  sha256: 2f235c432ef459970b4306369336b9d5dbdda31b510ca1e327636e01f528bfa9
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install rfc3986_validator-{{ version }}-py2.py3-none-any.whl --no-deps -vv"
+
+requirements:
+  build:
+  host:
+    - python
+    - setuptools
+    - pip
```

It seems the building phase needs ``pytest-runner``
```suggestion
    - pip
    - pytest-runner
```